### ISSUE TITLE
Script not working since WWDC2011?

### ISSUE THREAD

**Issue by and77** [tylerhall/sosumi#22]
Just politely asking:
I am having trouble getting data since 2011-06-06 10:30:00. That is the last data entry in my mySQL database.

Did something change? I surely didn't change anything on the weekend, but I fear Apple has been messing with a lot of stuff on Monday due to WWDC.
Anybody else having trouble getting the data?
Thanks!

**Comment by tylerhall** [comment]
It's broken again.

I'll have to look into it once I get back from WWDC.

**Comment by and77** [comment]
OK, thanks.
I hope you can fix it.

Enjoy the WWDC. ;-)

**Comment by tylerhall** [comment]
Sosumi is working again. Appears to have been something server side as I didn't have to change anything in Sosumi to make it work.

**Comment by and77** [comment]
Well, I had to change the app and build version in class.susomi.php from "1.1" to "1.2" and "99" to "145" respectively to get it back to work (and so did you.. I am guessing.).

Also I had to change "curl_setopt($ch, CURLOPT_FOLLOWLOCATION, false);" from "true" to "false", which might be an issue of my webspace.

To put it in a nutshell: I had to change a few things to get it back to work.

Thanks anyways! Still a great project!
